diff -u -r a/Makefile b/Makefile
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,6 @@
 TARGET          := $(shell uname -r)
 DKMS_ROOT_PATH  := /usr/src/zenpower-$(VERSION)

-KBUILD_CFLAGS   += -Wimplicit-fallthrough=3

 KERNEL_MODULES	:= /lib/modules/$(TARGET)

diff -u -r a/zenpower_core.c b/zenpower_core.c
--- a/zenpower_core.c
+++ b/zenpower_core.c
@@ -423,7 +423,7 @@
 			if (channel == 0)
 				return -EOPNOTSUPP;
 			channel -= 1;	// hwmon_in have different indexing, see note at zenpower_info
-							// fall through
+							/* fall through */
 		// Power / Current
 		case hwmon_curr:
 		case hwmon_power:
